templates/properties/lightblue-crud.json.erb

```diff
@@ -4,5 +4,22 @@
         "backend" : "mongo",
         "controllerFactory" : "com.redhat.lightblue.mongo.config.MongoCRUDFactory"
       }
+      
+      <% if additional_backend_controllers != '' && additional_backend_controllers.kind_of?(Array) -%>   
+        i = additional_backend_controllers.length
+        additional_backend_controllers.each do |controller|
+        ,   
+        {
+       "backend" : "<%=controller['backend']%>",
+       "controllerFactory" : "<%=controller['controllerFactory']%>"
+        }
+        
+        <% if i > 1 then -%>
```

Comment:
This logic around the comma needs removed.  It is already  included before the new backend is defined.  See line 11.